content/about/agreements.md

```diff
@@ -252,7 +252,8 @@ On the day of a lab members' birthday, all other members shall refer to them as
 We collectively co-create our community.  Being a part of this community affords us access to many amazing resources.  Associated with access to those resources and privileges are a number of responsibilities.
 
 - The privilege of being an official member of the team comes with the responsibility of 
-    - being physically present in the lab 2+ days/week for all full weeks in the academic calendar, modulo vacations, disease, or family emergencies, 
+    - being physically present on Monday and Thursday afternoon for all full weeks in the academic calendar, modulo classes, vacations, disease, or family emergencies,
```

Comment:
Are you sure we ended up deciding on monday and thursday afternoon? From what I remember, it was only Thursday at noon (for lab meeting).

Reply:
Oh really? My note told me for Monday (for meeting with Jovo)and Thursday afternoon(for lab meeting), but maybe you are right because Jovo doesn't require everyone to schedule weekly meetings with him.

Reply:
Honestly, I'm not sure what we ended up deciding regarding meeting responsibilities (meeting with jovo, small group meetings etc)... Let's bring it up in lab meeting this week to clarify.

Reply:
Great! Thank you again~